_topic_map.yml

```diff
@@ -22,259 +22,10 @@
 # topic groups and topics on the main page.
 
 ---
-Name: About
-Dir: welcome
-Distros: openshift-acs
+Name: Overview
```

Comment:
This makes the content appear as Overview > Overview

Maybe we can change it to:

- About > Overview
- Welcome > Overview